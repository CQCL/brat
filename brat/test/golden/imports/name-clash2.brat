import lib as A
import lib as B
import lib
open import lib
import lib (y)

main :: Nat, Int, Bool
main = x, y, z