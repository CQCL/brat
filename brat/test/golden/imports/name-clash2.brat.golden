Error in test/golden/imports/lib:
  Name clash: [lib.y]

